aws_promise/lambda.js

```diff
@@ -53,5 +53,22 @@ module.exports = {
       params['SourceArn'] = input.sourceArn;
     }
     return lambda.addPermission(params).promise();
+  },
+
+  invokeFunction: function(input) {
+    var lambda = this.findService(input);
+    var invocationType= (input.invocationType) ? input.invocationType : "RequestResponse";
+    var logType = (input.logType) ? input.logType : "Tail";
+    var qualifier= (input.qualifier) ? input.qualifier : "$LATEST";
+    var payload = JSON.stringify(input.payload);
+    var params = {
+      FunctionName: input.functionArn,
+      ClientContext: input.clientContext, 
+      InvocationType: invocationType, 
+      LogType: logType, 
+      Payload: new Buffer(payload, 'base64'), 
```

Comment:
pass payload as string only facing issues while passing it as buffer.

Reply:
Done.